src/NuGet.Core/NuGet.Packaging/PackagingConstants.cs

```diff
@@ -44,6 +44,18 @@ public static class Folders
                 Analyzers,
                 Source
             };
+
+            public static string[] SupportFrameworks { get; } = new string[] {
+                Build,
+                BuildCrossTargeting,
+                BuildTransitive,
+                Tools,
+                ContentFiles,
+                Lib,
+                Native,
```

Comment:
I've never heard of a `native/` package folder before, although I see it's listed in the `Known` array. Still, I'm not sure we ever look for `native/<tfm>/*`.